addons/trenches/CfgEventHandlers.hpp

```diff
@@ -30,3 +29,36 @@ class Extended_DisplayLoad_EventHandlers {
         ADDON = QUOTE(_this call COMPILE_FILE(XEH_missionDisplayLoad));
     };
 };
+
+class Extended_Deleted_EventHandlers {
+    class ACE_envelope_small {
+        class ADDON {
+            deleted = QUOTE(if (isServer) then {_this call FUNC(deleteCamouflage)});
+        };
+    };
+    class ACE_envelope_big: ACE_envelope_small {};
+    class ACE_envelope_giant: ACE_envelope_small {};
+    class ACE_envelope_vehicle: ACE_envelope_small {};
+    class ACE_envelope_short: ACE_envelope_small {};
```

Comment:
Since all of these inherit from `ACE_envelope_small`, you're adding this eventhandler twice.